src/main/java/Response/Response.java

```diff
@@ -0,0 +1,48 @@
+public class Response {
+
+  private String messageBody = "";
+  private String reasonPhrase = "";
+  private int statusCode;
+  private String version; 
+
+  public void setHTTPVersion(String version) {
+    this.version = version;
+  }
+
+  public void setStatusCode(int statusCode) {
+    this.statusCode = statusCode;
+  }
+
+  public void setReasonPhrase(String reasonPhrase) {
+    this.reasonPhrase = reasonPhrase;
+  }
+
+  public void setMessageBody(String messageBody) {
+    this.messageBody = messageBody;
+  }
+
+  public String getHTTPVersion() {
+    return this.version;
+  }  
+
+  public int getStatusCode() {
+    return this.statusCode;
+  }  
+
+  public String getReasonPhrase() {
+    return this.reasonPhrase;
+  }
+
+  public String getMessageBody() {
+    return this.messageBody;
+  }
+
+  public String getStatusLine() {
```

Comment:
This doesn't appear to be used anywhere else but the toString() method below

Reply:
I've removed this method.